Constitution.txt

```diff
@@ -1,490 +1,264 @@
-We the People of the United States, in Order to form a more perfect Union, establish Justice, insure domestic Tranquility, provide for the common defence, promote the general Welfare, and secure the Blessings of Liberty to ourselves and our Posterity, do ordain and establish this Constitution for the United States of America.
+We the People of these United States of America, in Order to form a more stable Union, establish Justice, ensure domestic Tranquility, provide for the common defense, and secure the Blessings of Liberty to ourselves and our Posterity, do ordain and re-establish this Constitution for these United States of America.
```

Comment:
I have updated all instances of "The United States" to "These United States," to emphasize the sovereignty of People as vested in the States, and recognize that they yield only elements of that sovereignty up to the federal government.

Reply:
There are also a number of spelling changes from linguistic drift. I've chosen to update some (like "ensure" and "choose"), and leave others, depending on whether I like them.

Reply:
I have removed the phrase "promote the general Welfare," because while the founders understood this to be a simple statement of purpose (that governments are only legitimate when the people find the existence of the state superior to anarchy), many people in the modern day would interpret this phrase to mean that the federal government should provide transfer payments to the impoverished. The authority to take such actions is not granted to the federal government in this constitution.